images/kcp-registrar/register.sh

```diff
@@ -26,6 +26,7 @@ usage() {
     printf "The following parameters need to be passed to the script:\n"
     printf "KCP_ORG: the organization for which the workload clusters need to be registered, i.e.: root:pipelines-service\n"
     printf "KCP_WORKSPACE: the name of the workspace where the workload clusters get registered (created if it does not exist), i.e: compute\n"
+    printf "KCP_TAG: the tag of the kcp syncer image to use (preset in the image when using Pipelines as Code)\n"
```

Comment:
We have to document KCP_TAG at this place as well.
https://github.com/openshift-pipelines/pipelines-service/blob/main/docs/kcp-registration.md